### PR TITLE
Add map overlay limits

### PR DESCRIPTION
**Pull request by rubenmoya**
Related to: #1716

screen shot 2017-08-31 at 11 10 31
screen shot 2017-08-31 at 11 10 35

### CODE REVIEW

File: src/geo/ui/overlays-view.js [hunk 1]
```diff
@@ -95,23 +93,34 @@ var OverlaysView = View.extend({
...
   },
 
   _getLoaderOverlay: function () {
     return this._getOverlayViewByType('loader');
   },
 
   _getOverlayViewByType: function (type) {
-    return _(this._overlayViews).find(function (v) {
-      return v.type === type;
+    return _.find(this._overlayViews, function (overlay) {
```
Comment (Jesus89) [inline src/geo/ui/overlays-view.js]: I think the "overlay" variable should be "view"

File: src/geo/ui/limits/limits-view.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  _onDocumentClick: function (event) {
+    if (!$(event.target).closest(this.el).length) {
+      this._toggleLimits();
+    }
+  },
+
+  _onVisibleChanged: function (_model, isVisible) {
+    isVisible ? this._showLimits() : this._hideLimits();
+  },
```
Comment (Jesus89) [inline src/geo/ui/limits/limits-view.js]: Tesing for this new view would be nice

File: src/geo/ui/limits/limits-view.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    return this;
+  },
+
+  _initBinds: function () {
+    this.model.on('change:visible', this._onVisibleChanged, this);
+  },
+
+  _enableDocumentBinds: function () {
+    $(document).bind('keydown', this._onDocumentKeyDown);
```
Comment (nobuti) [inline src/geo/ui/limits/limits-view.js]: Let's be consistent here: bind for context execution, on/off for events.

**Comment by nobuti** [comment]
Super nice work! 🚀

**Pull request merged**